packages/nx-playwright/src/generators/project/schema.json

```diff
@@ -39,7 +39,7 @@
     "packageRunner": {
       "enum": ["yarn", "npx", "pnpm"],
       "description": "package runner to use to run playwright",
-      "default": "yarn"
+      "default": "pnpm"
```

Comment:
Non-blocking: is it possible to have it default to nothing so that it by default inherits the package runner for the repo